### ISSUE TITLE
Disallow the usage of error control operator @

### ISSUE THREAD

**Issue by dingo-d** [WPTRT/WordPress-Coding-Standards#164]
Warning for the usage of the error control operator already exists in phpcs

> Silencing errors is discouraged (Generic.PHP.NoSilencedErrors.Discouraged)

We should set this to error if found in the theme, as I really don't see a place where and why this would be used.

#### Rule type:
Error - Rule should be set to error instead of warning, discouraging the users of using error silencing in the themes.

#### Rule:

> Rule as it is found in the handbook.

Handbook mentions that the usage of error control operator is discouraged.

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#error-control-operator

#### Decision needed:
Currently usage of error control operator is not advised, and is discouradged, although there is no specific WordPress sniff for it.

We should raise this to error status - I really cannot think of what would be the legitimate usage of it in the theme context.

##### Notes for implementation:
Basically, we should add this to the ruleset

```xml
<rule ref="Generic.PHP.NoSilencedErrors">
    <properties>
        <property name="error" value="true" />
    </properties>
</rule>
```

#### To do:
- [ ] Add the rule in the Theme Review handbook to the **_Requirements_** page.
- [ ] Create unit tests

**Comment by jrfnl** [comment]
Closed as duplicate of #83 which was solved by PRs #84 and #99
